frontend/src/components/VisaStatusOverviewList/OverviewContent.js

```diff
@@ -0,0 +1,263 @@
+    const qqTgInfoSelector = useMemo(() => makeQqTgInfoSelector(index), [index]);
+    const [qqGroups, tgLink] = useSelector(state => qqTgInfoSelector(state));
+    const [QQTGSubsModalVisible, setQQTGSubsModalVisible] = useState(false);
+    return (
+        <Space direction="horizontal">
+            <Tooltip title={t("overviewEmailIcon")}>
+                <Button icon={<MailOutlined />} shape="circle" onClick={() => onEmailSubsClick()} />
+            </Tooltip>
+            {isVisaTypeF && (
+                <Tooltip title={t("overviewQQIcon")}>
+                    <Button icon={<QqOutlined />} shape="circle" onClick={() => setQQTGSubsModalVisible(true)} />
+                </Tooltip>
+            )}
+            <Tooltip title={t("overviewAddtionalIcon")}>
+                <Button icon={<EllipsisOutlined rotate={90} />} shape="circle" onClick={() => onAddtionClick()} />
+            </Tooltip>
+            {isVisaTypeF && (
+                <QQTGSubsModal
+                    qqGroups={qqGroups}
+                    tgLink={tgLink}
+                    isModalVisible={QQTGSubsModalVisible}
+                    setIsModalVisible={setQQTGSubsModalVisible}
+                />
+            )}
+            {children}
+        </Space>
+    );
+};
+ContentActions.propTypes = {
+    children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
+    embassyCode: PropTypes.string.isRequired,
+    onEmailSubsClick: PropTypes.func.isRequired,
+    onAddtionClick: PropTypes.func.isRequired,
+};
+
+const ContentBar = ({ embassyCode, earliestDate, latestDate, newest }) => {
+    const [t] = useTranslation();
+    const { writeTime, availableDate } = newest;
+    return (
+        <Row align="middle" className="ovreview-content-row" gutter={16}>
+            <Col md={{ span: 3 }}>{t(embassyCode)}</Col>
+            <Col md={{ span: 4 }}>
+                <Tooltip title={t("overviewEarliest")}>{earliestDate.join("/")}</Tooltip>
+            </Col>
+            <Col md={{ span: 4 }}>
+                <Tooltip title={t("overviewLatest")}>{latestDate.join("/")}</Tooltip>
+            </Col>
+            <Col md={{ span: 8 }}>
+                <Tooltip title={t("overviewNewest")}>
+                    <Row justify="center" align="middle">
+                        <Col xs={{ span: 11 }} md={{ span: 9 }}>
+                            {availableDate.join("/")}
+                        </Col>
+                        <Col xs={{ span: 13 }} md={{ span: 9 }}>
+                            <Tag>{`${t("at")} ${
+                                writeTime.length === 1 ? writeTime : getTimeFromUTC(writeTime).join(":")
+                            }`}</Tag>
+                        </Col>
+                    </Row>
+                </Tooltip>
+            </Col>
+            <Col md={{ span: 5 }}>
+                <ContentActions embassyCode={embassyCode} onEmailSubsClick={() => {}} onAddtionClick={() => {}} />
+            </Col>
```

Comment:
Since the action button group will be responsive to the `visaType` I suggest we justify the button group align to the end, so the button group extend only toward left, instead of extending from middle to both side.

```js
<Col md={{ span: 5 }}>
    <Row justify="end">
        <Col>
            <ContentActions embassyCode={embassyCode} onEmailSubsClick={() => {}} onAddtionClick={() => {}} />
        </Col>
    </Row>
</CoL>
```